corehq/form_processor/change_publishers.py

```diff
@@ -80,3 +93,11 @@ def change_meta_from_ledger_v1(stock_state):
         domain=stock_state.domain,
         is_deletion=False,
     )
+
+
+def _get_cases_from_form(domain, form):
+    processor = FormProcessorInterface(domain=domain)
+    case_updates = processor.get_cases_from_forms(
+        case_db=processor.casedb_cache(domain=domain), xforms=[form]
+    ).values()
```

Comment:
Is it necessary to call this method to get the cases? It seems overkill to me since it will load them from the DB and also apply all the updates from the form. Could we not just do this:
```
case_ids = case_ids = get_case_ids_from_form(form) | get_case_ids_from_stock_transactions(form)
cases = FormAccessors(domain).get_cases(list(case_ids))
```